examples/all-clusters-app/cc13x2x7_26x2x7/main/AppTask.cpp

```diff
@@ -30,13 +30,8 @@
 
 #include <platform/CHIPDeviceLayer.h>
 
-#include <app/clusters/ota-requestor/BDXDownloader.h>
```

Comment:
in case someone would like to re-enable this don't you think a compile flag would have been a good idea ?
or we absolutely don't care for this.... ?

Reply:
Lets resolve this as a separate conversation - want to merge this because ToT is red in CI.